core/tests/protractor_desktop/topicAndStoryEditor.js

```diff
@@ -348,20 +336,17 @@ describe('Chapter editor functionality', function() {
 
   it('should fail to add one more chapter with existing exploration',
     async function() {
-      await storyEditorPage.navigateToChapterByIndex(1);
-      await storyEditorPage.createNewChapter('Chapter 3');
-      await storyEditorPage.navigateToChapterByIndex(2);
-      await storyEditorPage.setChapterExplorationId(dummyExplorationIds[1]);
-      await storyEditorPage.expectExplorationIdAlreadyExistWarningAndCloseIt();
-      allowedErrors.push('The given exploration already exists in the story.');
+      await storyEditorPage.createNewChapter(
+        'Chapter 3', dummyExplorationIds[1], '../data/test_svg.svg');
+      await storyEditorPage.expectExplorationIdAlreadyExistWarning();
     }
   );
 
   it('should add one more chapter and change the chapters sequences',
     async function() {
-      await storyEditorPage.createNewChapter('Chapter 3');
+      await storyEditorPage.createNewChapter(
+        'Chapter 3', dummyExplorationIds[2], '../data/test_svg.svg');
```

Comment:
Does this work in prod mode even though it involves uploading the SVG?

Reply:
I am not sure what you mean? 
The feature works in prod mode. Also this e2e test is part of the non-prod section right? @U8NWXD

Reply:
Yep, you're absolutely right. Please disregard my comment